.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] **I am adding a new extension to an existing language.**
+  - [ ] The new extension is used in hundreds of repositories on GitHub.com
+    - Search results for each extension:
+      <!-- Replace FOOBAR with the new extension, and KEYWORDS with keywords unique to the language. Repeat for each extension added. -->
+      - https://github.com/search?utf8=%E2%9C%93&q=extension%3AFOOBAR+KEYWORDS+NOT+nothack&type=Code&ref=searchresults
```

Comment:
It would help to move the `FOOBAR+KEYWORDS` to the end of the URL, for easier copy+pasta. Although I find it amazing there's no front-end mechanism for running this search, yet there's a list of language colours buried away in https://github.github.com/linguist/ 😣